ci/helm/preview/values.yaml

```diff
@@ -43,7 +43,7 @@ nuxeo:
       pullPolicy: Always
     persistence:
       enabled: disable
-    packages: nuxeo-web-ui-2.4.46-NAPPS-SNAPSHOT amazon-s3-online-storage ${JSF_ENABLED} /packages/nuxeo-coldstorage-package.zip
+    packages: nuxeo-web-ui-2.4.0 amazon-s3-online-storage ${JSF_ENABLED} /packages/nuxeo-coldstorage-package.zip
```

Comment:
can you please ask the WebUI team if they have released a new version for 10.10

Reply:
if it’s the case, we should remove this line `org.nuxeo.connect.url=https://nos-preprod-connect.nuxeocloud.com/nuxeo/site/`

Reply:
as we discussed, we'll wait for https://qa2.nuxeo.org/jenkins/view/10.10/job/10.10/job/plugins_nuxeo-web-ui-2.4_10.10/ to pass before changing the version. It will stay a pre-prod, but we still don't know the version name, probably something like nuxeo-web-ui-2.4.47